.travis.yml

```diff
@@ -37,6 +37,12 @@ matrix:
     dist: bionic
     compiler: gcc
     name: "Ubuntu 18.04 (C++)" 
+  - os: linux
+    dist: bionic
```

Comment:
I think we can remove the travis check since  .github/workflows/rat.yml already does the check.

Reply:
Yes, travis check is removed. This PR can be a first step in moving from travis to github actions. Then later we can move linting and other scripts also.
Can we merge it directly to master? or we still need to merge it with dev first?